src/rem/UpgradeOptions.tsx

```diff
@@ -88,7 +90,7 @@ export const UpgradeOptions: FC<{
         aria-labelledby={labelId}
         aria-describedby={descriptionId}
       >
-        <div className="flex w-full justify-between">
+        <div className="flex justify-between w-full">
```

Comment:
Are you using some Tailwind class sorting plugin? (This change is fine, I'm just wondering if we should introduce something like that in this codebase)

Reply:
Ah, yes. This is a [Prettify plugin](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier). Must have it in a global setting in VS Code.